### DISCUSSION TITLE
New receiver method on `*pg.Options` `ToURL`?

### DISCUSSION THREAD

**Discussion by dillonstreator** [go-pg/pg#1933] [Ideas]
The `pg.ParseURL` method accepts a url and returns `*pg.Options`
Is there a way to go from `*pg.Options` back to the url?

For instance, I'd like to get the options from the url, mutate options, and then get the updated url from the mutated options.

Maybe a receiver method on `*pg.Options`?
```go
func (o *Options) ToURL() string {
    // ...
}
```

**Comment by elliotcourant** [comment]
I'd be up for trying to implement such a method.

**Reply by dillonstreator** [reply]
Ah just saw your response! I actually have a PR up and would love some feedback https://github.com/go-pg/pg/pull/1934
